CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## 1.17.2
+* Enhancement - Expose two more metadata fields, task definition family and task definition revision [#1295](https://github.com/aws/amazon-ecs-agent/pull/1295)
```

Comment:
This should be under `1.17.3-dev`. You'll see it if you rebase it.

Reply:
OK, thanks for pointing out, I just rebased and updated the change log again